snapd.go

```diff
@@ -221,6 +221,10 @@ func action(ctx *cli.Context) {
 	// same variables in that configuration
 	applyCmdLineFlags(cfg, ctx)
 
+	// Switch log level to user defined
+	log.Info("setting log level to: ", l[cfg.LogLevel])
+	log.SetLevel(getLevel(cfg.LogLevel))
```

Comment:
Should we add the log.Info statement after setting the level so if they have specified a level > info they won't see this info statement?

 Also if you specify a log path this line will be printed to stdout while the rest goes to the logpath.